typings/d3/d3.d.ts

```diff
@@ -1674,7 +1674,7 @@ declare module D3 {
         export interface Symbol {
             type: (symbolType: string | ((datum: any, index: number) => string)) => Symbol;
             size: (size: number | ((datum: any, index: number) => number)) => Symbol;
-            (datum:any, index:number): string;
+            (datum?: any, index?: number): string;
```

Comment:
Is there a PR going to DefinitelyTyped with this, or did we just upgrade the typing file?